noir-projects/noir-contracts/contracts/test/counter_contract/src/test.nr

```diff
@@ -1,19 +1,24 @@
 use crate::Counter;
-use dep::aztec::{
-    protocol_types::address::AztecAddress, test::helpers::test_environment::TestEnvironment,
+use aztec::{
+    oracle::notes::set_sender_for_tags, protocol_types::address::AztecAddress,
+    test::helpers::test_environment::TestEnvironment,
 };
 
 pub unconstrained fn setup(
     initial_value: Field,
-) -> (&mut TestEnvironment, AztecAddress, AztecAddress, AztecAddress) {
+) -> (&mut TestEnvironment, AztecAddress, AztecAddress) {
     // Setup env, generate keys
     let mut env = TestEnvironment::new();
     let owner = env.create_account(1);
-    let sender = env.create_account(2);
+
+    // We are not deploying any account contracts and so we need to manually set the sender for tags. This is usually
+    // done by account contract's entrypoint.
+    let sender = env.create_account(7097070);
+    set_sender_for_tags(sender);
```

Comment:
Based on Nico's TXE refactor I am aware that we shouldn't directly call oracles in tests. But since we currently don't have `set_sender_for_tags` exposed on `TestEnvironment` think it makes sense to do in the future.